### DISCUSSION TITLE
Is there anyway to delete a file link?

### DISCUSSION THREAD

**Discussion by chinguetti** [O365/python-o365#746] [Q&A]
I have hit the microsoft limit on shared files for my account.  Is there any way to remove a link from a file?

MS graph API documentation is here:
https://docs.microsoft.com/en-us/graph/api/permission-delete?view=graph-rest-1.0&tabs=http

If a feature is in the native graph API but not implemented in python-o365 is there a way we can call it directly?

To the developers of this package, a huge thank you!  You are legends, seriously.

**Comment by alejcas** [comment]
> If a feature is in the native graph API but not implemented in python-o365 is there a way we can call it directly?

Yes but you will have to implement that call yourself. PR are welcome!

Thanks!
